themes/default/content/registry/packages/aws/how-to-guides/bucketv2-migration.md

```diff
@@ -16,17 +16,28 @@ To migrate existing `aws.s3.Bucket` resources to `aws.s3.BucketV2`:
    `BucketV2` inputs such as `policy` and `accelerationStatus` are to be replaced by side-by-side resources
    `aws.s3.BucketPolicy` and `aws.s3.BucketAccelerateConfigurationV2`.
 
-2. Perform `pulumi up` to replace the buckets in AWS. **WARNING**: replacing the buckets will delete and re-create them,
-   losing any data stored in these buckets. If this is not acceptable, consider using `pulumi import` to migrate
+2. Perorm this is not acceptable, consider using `pulumi import` to migrate
```

Comment:
Maybe something like

"In order to avoid resource replacement, consider using `pulumi import` to manually migrate resources"

Reply:
Hmm, I think I have spellechecker in Emacs but I missed this, let me double-check.